### PR TITLE
Foundation styles & and other styles

### PR DESCRIPTION
**Pull request by lschatzkin**
Responsive & other fun stuff.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,52 +12,13 @@
...
     </div>
 
-  <footer>
-    <div class="row">
-      Footer
-    </div>
-  </footer>
-  <%= javascript_include_tag "application" %>
-
-  <div id="anon-stats">
```
Comment (wioux) [inline app/views/layouts/application.html.erb]: Can you restore this div? As well as the `#anon-stats` rule from layout.scss.

Comment (lschatzkin) [reply app/views/layouts/application.html.erb]: Sorry about that! I got delete happy when removing the old footer div/styles.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -19,6 +19,7 @@
     </div>
 
 <%= render "layouts/footer" %>
-
+<div id="anon-stats">
```
Comment (wioux) [inline app/views/layouts/application.html.erb]: It should be like this:

https://github.com/EFForg/sec/blob/d7120ce2b520e5ddc6873a20b52e78532a566457/app/views/layouts/application.html.erb#L53-L60

**Comment by vbrown608** [comment]
This looks great! Thanks for digging the templating stuff :D I think it's ready for merge once conflicts are resolved.

**Pull request merged**